platforms/nodejs/versions/bullseye/versionsToBuild.txt

```diff
@@ -7,7 +7,6 @@
 12.22.9
 12.22.11
 12.22.12
-
```

Comment:
nit: any reason we removed newlines from these versionsToBuild text files?

Reply:
Removing these so we can sort the versionsToBuild file whenever a new item gets added